internal/k8sutils/cluster-scaling.go

```diff
@@ -63,19 +65,20 @@ func ReshardRedisCluster(ctx context.Context, client kubernetes.Interface, cr *r
 	cmd = append(cmd, transferNodeID)
 
 	// Cluster Slots
-	slot := getRedisClusterSlots(ctx, redisClient, removeNodeID)
+	slots := getRedisClusterSlots(ctx, redisClient, removeNodeID)
+	if slots == "0" || slots == "" {
+		log.FromContext(ctx).Info("skipped the execution of reshard command since the slots is empty", "Cmd", cmd, "Slots", slots)
+		return
+	}
 	cmd = append(cmd, "--cluster-slots")
-	cmd = append(cmd, slot)
+	cmd = append(cmd, slots)
 
 	cmd = append(cmd, "--cluster-yes")
 
-	log.FromContext(ctx).V(1).Info("Redis cluster reshard command is", "Command", cmd)
-
-	if slot == "0" {
-		log.FromContext(ctx).V(1).Info("Skipped the execution of", "Cmd", cmd)
-		return
-	}
-	executeCommand(ctx, client, cr, cmd, cr.ObjectMeta.Name+"-leader-0")
+	log.FromContext(ctx).V(1).Info("redis cluster reshard command is", "Command", cmd)
+	log.FromContext(ctx).Info("transferring slots", "Slots", slots, "from", shardIdx, "to", transferNodeIdx)
```

Comment:
@drivebyer I was considering whether to log it this way but tbh I find the logs much harder to read when they a broken down into separate, unordered, map keys. 
Am I missing some obvious way the logs should be consumed ? I'm just doing `kubectl logs`

Reply:
It's more structured and better for log system such as ES.

Reply:
Fair; in my experience with structured logs, I've found it more useful to have the logged message itself be complete and then the other fields to be treated as an addition to it. 

E.g. here it could be `log.FromContext(ctx).V(0).Info(fmt.Sprintf("Transferring %s slots from shard %d to shard %d", slots, shardIdx, transferNodeIdx), "from_shard", shardIdx, "to_shard", transferNodeIdx)`. Happy to make the change if you want, that way it will be more consistent with the codebase (although it would be a bit repetitive)